src/Controller/GameController.php

```diff
@@ -73,7 +80,12 @@ public function newGame(request $request): Response
         $entityManager->persist($game);
         $entityManager->flush();
 
-        $response = new JsonResponse(new GameDto($game));
+
+        $response = new Response(
```

Comment:
why no longer use the `JsonResponse`? Doesn't it avoid having to declare the `['Content-type' => 'application/json']` for example?

Reply:
Because JsonReponse handle an associative array to convert it as JSON.

The JSON serializer did the same, and if I use both, response of my API will be stringified JSON.
But I could try something else.

Reply:
Using JsonResponse::fromJsonString allow to handle directly Json format.

Others response are now correctly transformed.

`Content-Type` is now implicit by using `JsonResponse`